src/main/webapp/scripts/forum-script.js

```diff
@@ -1,7 +1,17 @@
 import {forumTemplate} from './forum-template.js';
 
-$( document ).ready( getFilters );
-$( document ).ready( getForum );
+$( document ).ready( function() {
+  getForum();
+  getFilters();
+  $('#search-button').click(search);
+  $('#search-button').keyup(function(event) {
+    const enterKey = 13;
+    if (event.keyCode === enterKey) {
```

Comment:
Is it very slow if you call `search` for *every* key press? It could be nice to have it be a "live" search box that updates on every keypress.

Reply:
Its not too slow it seems.  I updated this.